src/main/java/org/assertj/core/error/ShouldHaveAllNullFields.java

```diff
@@ -0,0 +1,48 @@
+package org.assertj.core.error;
+
+import java.util.List;
+
+/**
+ * Creates an <code>{@link AssertionError}</code> indicating that an assertion that verifies that an object
+ * has null fields failed.
+ *
+ * @author Vladimir Chernikov
+ */
+public class ShouldHaveAllNullFields extends BasicErrorMessageFactory {
+
+  private static final String EXPECTED_MULTIPLE = "%nExpecting%n  <%s>%nto have properties or fields named <%s> are null.%n";
+  private static final String EXPECTED_SINGLE = "%nExpecting%n  <%s>%nto have null property or field, but <%s> was not null.%n";
```

Comment:
replace `to have null property or field` by `to only have null properties or fields`

Reply:
Changed.